src/cheri/introduction.adoc

```diff
@@ -30,115 +30,123 @@ be brute forced.
-
-We refer to software as _purecap_ if it utilizes CHERI capabilities for all
-memory accesses -- including loads, stores and instruction fetches -- rather
-than integer addresses. Purecap software requires the CHERI RISC-V hart to
-support {cheri_base_ext_name}. We refer to software as _hybrid_ if it uses
-integer addresses *or* CHERI capabilities for memory accesses. Hybrid software
-requires the CHERI RISC-V hart to support {cheri_base_ext_name} and
-{cheri_default_ext_name}.
-
-See xref:app_cheri_instructions[xrefstyle=short] for compatibility with other RISC-V
-extensions.
-
-=== Risks and Known Uncertainty
+^1^ {cheri_priv_crg_load_tag_ext} is available for improved software revocation performance if {cheri_priv_crg_ext} is implemented.
 
-* All extensions could be divided up differently in the future, including after
-ratification
-* The RISC-V Architecture Review Committee (ARC) are likely to update all
-encodings
-* The ARC are likely to update all CSR addresses
-* Instruction mnemonics may be renamed
-    ** Any changes will affect assembly code, but assembler aliases can provide
-backwards compatibility
+.Debug stable extensions and specifications
+[#debug-extension-status,reftext="Extension Status and Summary"]
+[options=header,align=center,width="90%"]
+|=============================================================================================================================================================
+| Extension                                                 | Description
+|<<section_debug_integration_ext,Sdext ({cheri_base_ext_name})>>   | External debug support
+|=============================================================================================================================================================
 
-==== Partially Incompatible Extensions
+==== Experimental Extensions and Specifications
 
-There are RISC-V extensions in development that may duplicate some aspects of
-CHERI functionality or directly conflict with CHERI and should only be
-available in {cheri_int_mode_name} on a CHERI-enabled hart.
-These include:
+.Unprivileged experimental extensions and specifications
+[#unpriv-exp-extension-status,reftext="Extension Status and Summary"]
+[options=header,align=center,width="90%"]
+|=============================================================================================================================================================
+| Extension                                                 | Description
+|<<section_cheri_vector_integration>>                       | Vector
+|<<Zcmt_CHERI>>                                             | Table Jump for {cheri_base32_ext_name}
+|<<Zcmp_CHERI>>                                             | Push/pop and double move for {cheri_base32_ext_name}
+|<<section_xycheriot>>                                      | CHERIoT capability encoding
+|<<section_zyseal>>                                         | Sealing extension required for CHERIoT
```

Comment:
It happens to be required for CHERIoT, but it is useful unto itself.  It's been in University CHERI for over a decade.